runtime/gc_glue_java/ObjectModel.hpp

```diff
@@ -558,6 +558,28 @@ class GC_ObjectModel : public GC_ObjectModelBase
 		return adjustSizeInBytes(getObjectModelDelegate()->getObjectSizeInBytesWithHeader(objectPtr, hasBeenMoved(objectPtr) && !hasRecentlyBeenMoved(objectPtr)));
 	}
 
+	MMINLINE void
+	calculateObjectDetailsForCopy(MM_EnvironmentBase *env, MM_ForwardedHeader *forwardedHeader, uintptr_t *objectCopySizeInBytes, uintptr_t *objectReserveSizeInBytes, uintptr_t *hotFieldAlignmentDescriptor)
+	{
+		GC_ObjectModelBase::calculateObjectDetailsForCopy(env, forwardedHeader, objectCopySizeInBytes, objectReserveSizeInBytes, hotFieldAlignmentDescriptor);
+	}
```

Comment:
In order to migrate the below `calculateObjectDetailsForCopy` I had to create this, if not compiler complains that this method signature doesn't exist (from Scavenger)

Reply:
or to avoid this maybe the other (new) one can just go to GC_ObjectModelBase?

Reply:
True, so that will need to go in the OMR PR. So in order not to break anything, maybe we can introduce them here in this PR, then introduce the new `calculateObjectDetailsForCopy` in GC_ObjectModelBase in OMR PR; and then we can remove these from `ObjectModel.hpp` in a third OpenJ9 PR.

Reply:
These sequence of changes are inevitable: OpenJ9 PR -> OMR PR -> OpenJ9 PR. Therefore your suggestion would be introduced that way

Reply:
sounds good